bofire/strategies/predictives/botorch.py

```diff
@@ -267,6 +293,85 @@ def _postprocess_candidates(self, candidates: Tensor) -> pd.DataFrame:
         preds = self.predict(df_candidates)
         return pd.concat((df_candidates, preds), axis=1)
 
+    def _optimize_acqf_continuous(
+        self,
+        candidate_count: int,
+        acqfs: List[AcquisitionFunction],
+        bounds: Tensor,
+        ic_generator: Callable,
+        ic_gen_kwargs: Dict,
+        nchooseks: List[Callable[[Tensor], float]],
+        fixed_features: Optional[Dict[int, float]],
```

Comment:
What are `fixed_features` and `fixed_features_list`? Are these features not to be optimized? Why are there two of those? For batch proposals?

Reply:
`fixed_features` is used in  botorchs `optimize_acqf` when you just optimize over a continuous space. It is a dictionary that maps indices to float values. The indices mark the features that should be kept fixed. And the floats are the respective fixed values. In case that you optimize over a mixed space using `optimize_acqf_mixed`, you have to provide a list of fixed_features dictionaries. It is then running an optimization for every dictionary in the list and returns the best one. It is documented here: https://github.com/pytorch/botorch/blob/b5b4fb7c316983f0cfb2f3257247e52c71a23b7d/botorch/optim/optimize.py#L417 and here https://github.com/pytorch/botorch/blob/b5b4fb7c316983f0cfb2f3257247e52c71a23b7d/botorch/optim/optimize.py#L855

Reply:
Thanks for the Botorch-Nachhilfe :)